openid-4-verifiable-presentations-1_0.md

```diff
@@ -261,6 +261,9 @@ This specification defines the following new parameters:
 
 A public key to be used by the Wallet as an input to the key agreement to encrypt Authorization Response (see (#jarm)). It MAY be passed by the Verifier using the `jwks` or the `jwks_uri` claim within the `client_metadata` or `client_metadata_uri` request parameter.
 
+`create_request_uri`: 
+: OPTIONAL. A string containing an HTTPS URL pointing to a resource where the Wallet MUST request the creation of a request object as defined in [@RFC9101]. The details of this endpoint are defined in (#create_request_uri). This parameter MUST only be combined with `client_id`, `state`, `client_id_scheme`, and any client id scheme specific parameter. It SHOULD be sent in a signed authorization request. 
+
 The following additional considerations are given for pre-existing Authorization Request parameters:
 
 `nonce`:
```

Comment:
IMO, `nonce` should be provided only in the`create_request_uri` response which is a (signed) authorization request object in case `create_request_uri` is provided. Otherwise, it is fine to pass the nonce in the complete JAR in the first step.